src/librustc/lint/context.rs

```diff
@@ -155,20 +155,24 @@ impl LintStore {
             .collect()
     }
 
-    pub fn register_early_pass(&mut self, pass: fn() -> EarlyLintPassObject) {
-        self.early_passes.push(pass);
+    pub fn register_early_pass(&mut self,
```

Comment:
Nit: put `&mut self` on next line, to align with rustfmt? (Similarly, the `) {` on the next line)